scripts/release.patch.rst

```diff
@@ -7,9 +7,9 @@ This is a bug-fix release, being a drop-in replacement. To upgrade::
 
   pip install --upgrade pytest
 
-The full changelog is available at https://docs.pytest.org/en/latest/changelog.html.
+The full changelog is available at https://docs.pytest.org/en/stable/changelog.html.
 
-Thanks to all who contributed to this release, among them:
+Thanks to all contributors of this release:
```

Comment:
I'd probably say "Thanks to all of the contributors to this release:"

Reply:
Better, thanks!